cranelift/codegen/src/ir/entities.rs

```diff
@@ -106,7 +106,7 @@ entity_impl!(Inst, "inst");
 /// [call stack](https://en.wikipedia.org/wiki/Call_stack).
 ///
 /// `StackSlot`s can be created with
-/// [`FunctionBuilder::create_stackslot`](https://docs.rs/cranelift-frontend/*/cranelift_frontend/struct.FunctionBuilder.html#method.create_stack_slot).
+/// [`FunctionBuilder::create_sized_stack_slot`](https://docs.rs/cranelift-frontend/*/cranelift_frontend/struct.FunctionBuilder.html#method.create_sized_stack_slot).
```

Comment:
Can we mention `create_dynamic_stack_slot` here too? (Both this and the sized variant are peers -- both are valid ways to create stackslots, for different sorts of types.)

Reply:
Sure! Thanks for the heads up. I've re-pushed the commit with the requested changes.